ui/desktop/app/controllers/application.js

```diff
@@ -15,14 +17,34 @@ export default class ApplicationController extends Controller {
   @service clusterUrl;
   @service flashMessages;
 
+  // =tracked properties
+
+  @tracked showSignoutModal = false;
+  @tracked showCloseSessionsModal = false;
+
+  constructor() {
+    super(...arguments);
+
+    // Check if Electron and ipcRenderer are available
+    if (window.electron && window.electron.ipcRenderer) {
+      window.electron.ipcRenderer.on('inform-ember-of-app-closure', () => {
```

Comment:
Can we call the handler something like `onAppQuit`?

Reply:
Yep! sure can